src/components/suggest/suggest-list-item/ch-suggest-list-item.scss

```diff
@@ -6,4 +6,5 @@
   display: flex;
   width: 100%;
   cursor: pointer;
+  box-sizing: border-box;
```

Comment:
It's best to ensure that all elements have this property by including the `box-sizing()` mixin.
Usage example: https://github.com/genexuslabs/chameleon-controls-library/blob/main/src/components/textblock/textblock.scss#L3